ios/GoodDollar/RNPushNotificationIOS+AppDelegate.m

```diff
@@ -0,0 +1,48 @@
+//
+//  RNPushNotificationIOS+AppDelegate.m
+//  GoodDollar
+//
+//  Created by Nicolas Castellanos on 2/28/20.
+//  Copyright © 2020 Facebook. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "AppDelegate.h"
+#import <RNCPushNotificationIOS.h>
+#import <UserNotifications/UserNotifications.h>
+
+@implementation AppDelegate(AppDelegate)
+
+// Required to register for notifications
+- (void)application:(UIApplication *)application didRegisterUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings
```

Comment:
Those methods are deprecated in the UIApplicationDelegate protocol.
The UNUserNotificationCenterDelegate  should be used.

Please update to the latest push-notification-ios and react-native-push-notifications, reconfigure your ios app according to the latest docs

Reply:
I updated following this documentation https://github.com/react-native-community/push-notification-ios